.travis.yml

```diff
@@ -7,37 +7,40 @@ cache:
         - vendor
         - $HOME/.composer/cache/files
 
-php:
-    - 5.4
-    - 5.5
-    - 5.6
-    - 7.0
-    - 7.1
-    - 7.2
-    - 7.3
-
 env:
     - TWIG_EXT=no
+    - DRUPAL=no
 
 before_install:
-    # turn off XDebug
     - phpenv config-rm xdebug.ini || return 0
 
 install:
     - travis_retry composer install
+    - wget https://get.symfony.com/cli/installer -O - | bash
 
 before_script:
     - if [ "$TWIG_EXT" == "yes" ]; then sh -c "cd ext/twig && phpize && ./configure --enable-twig && make && make install"; fi
     - if [ "$TWIG_EXT" == "yes" ]; then echo "extension=twig.so" >> `php --ini | grep "Loaded Configuration" | sed -e "s|.*:\s*||"`; fi
 
-script: ./vendor/bin/simple-phpunit
+script: |
+    ./vendor/bin/simple-phpunit
+    if [ "$DRUPAL" == "yes" ]; then ./drupal_test.sh; fi
```

Comment:
please do that with 2 separate entries for `script` (it supports arrays, like other sections), to be sure to have proper error handling from Travis here.

Reply:
done